cmd/zetae2e/local/precompiles.go

```diff
@@ -0,0 +1,56 @@
+package local
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/fatih/color"
+
+	"github.com/zeta-chain/zetacore/e2e/config"
+	"github.com/zeta-chain/zetacore/e2e/e2etests"
+	"github.com/zeta-chain/zetacore/e2e/runner"
+)
+
+// erc20TestRoutine runs erc20 related e2e tests
+func statefulPrecompilesTestRoutine(
+	conf config.Config,
+	deployerRunner *runner.E2ERunner,
+	verbose bool,
+	testNames ...string,
+) func() error {
+	return func() (err error) {
+		account := conf.AdditionalAccounts.UserERC20
```

Comment:
Reason we define new account is because the tests run in parallel, if same account send tx at the same time there might be issue with nonce, etc..
I think we can create a new account for Precompile interactions